21.1.0 has broken typings

```
node_modules/@types/yargs/index.d.ts(27,10): error TS2305: Module '"yargs-parser"' has no exported member 'DetailedArguments'.
node_modules/@types/yargs/index.d.ts(27,29): error TS2305: Module '"yargs-parser"' has no exported member 'Configuration'.
```

Version posted 30 minutes ago does not work

Happen to me using ts-jest that has yargs-parser : ^21.0.0 as dependency 

`09:36:32  node_modules/@types/yargs/index.d.ts(28,10): error TS2305: Module '"yargs-parser"' has no exported member 'DetailedArguments'.
09:36:32  node_modules/@types/yargs/index.d.ts(28,29): error TS2305: Module '"yargs-parser"' has no exported member 'Configuration'.`

> Happen to me using ts-jest that has yargs-parser : ^21.0.0 as dependency
> 
> `09:36:32 node_modules/@types/yargs/index.d.ts(28,10): error TS2305: Module '"yargs-parser"' has no exported member 'DetailedArguments'. 09:36:32 node_modules/@types/yargs/index.d.ts(28,29): error TS2305: Module '"yargs-parser"' has no exported member 'Configuration'.`

Same here.

--- a/package.json
+++ b/package.json
@@ -68,7 +68,7 @@
     "rimraf": "^3.0.2",
     "rollup": "^2.22.1",
     "rollup-plugin-cleanup": "^3.1.1",
-    "rollup-plugin-ts": "^3.0.0",
+    "rollup-plugin-ts": "^3.0.2",
     "serve": "^14.0.0",
     "standardx": "^7.0.0",
     "start-server-and-test": "^1.11.2",
@@ -78,7 +78,8 @@
   "files": [
     "browser.js",
     "build",
-    "!*.d.ts"
+    "!*.d.ts",
+    "!*.d.cts"
   ],
   "engines": {
     "node": ">=12"

